src/api/app/views/webui/request/_request_history_element.html.haml

```diff
@@ -8,6 +8,9 @@
         = link_to(element.user_action, request_show_path(element.description_extension))
       - else
         = element.user_action
+      - if element.respond_to?(:review) && element.review
+        for
+        = render partial: 'webui/request/reviewer', locals: { review: element.review }
```

Comment:
I think you can work on the design of this a bit

![Screenshot from 2019-09-19 17-00-36](https://user-images.githubusercontent.com/514785/65255968-0f511b00-daff-11e9-9ab9-d77eeedeee2f.png)

The click targets are very close together, the reviewer avatar looks out of place, the sentence always wraps etc.

Reply:
On it

Reply:
![image](https://user-images.githubusercontent.com/2650/65336653-117eac80-dbc7-11e9-80b4-8941b23064f2.png)
What about this?

Reply:
that's neatly faked user names. Can you try https://build.opensuse.org/request/show/732250's "factory-auto added review for opensuse-review-team 2 days ago" and "staging-bot added review for openSUSE:Factory:Staging:adi:74 2 days ago" in the little place you got there?

Reply:
Yeah, will try. Thanks!